Formula/luaradio.rb

```diff
@@ -0,0 +1,47 @@
+class Luaradio < Formula
+  desc "lightweight, embeddable flow graph signal processing framework for SDR"
+  homepage "http://luaradio.io/"
+  url "https://github.com/vsergeev/luaradio/archive/v0.1.0.tar.gz"
+  sha256 "4423f94c0bb259952eae7db11e4db4509dfa7c6e205798c35a095050c530212b"
+
+  depends_on "pkg-config" => :build
+  depends_on "luajit"
+  depends_on "fftw" => :recommended
+
+  def install
+    cd "embed" do
+      system "make", "install", "PREFIX=#{prefix}"
+    end
+  end
+
+  test do
+    # Create text file which will be read and processed by LuaRadio
```

Comment:
I'm serious. Not because of a comment but because of the process. The way Homebrew PRs work is we request you make changes and you either make our changes as requested or close the PR.